pandas/util/_validators.py

```diff
@@ -79,19 +74,18 @@ def validate_args(fname, args, max_fname_arg_count, compat_args):
     has at most `len(compat_args)` arguments and whether or not all of these
     elements in `args` are set to their default values.
 
-    fname: str
+    Parameters
+    ----------
+    fname : str
         The name of the function being passed the `*args` parameter
-
-    args: tuple
+    args : tuple
         The `*args` parameter passed into a function
-
-    max_fname_arg_count: int
+    max_fname_arg_count : int
         The maximum number of arguments that the function `fname`
         can accept, excluding those in `args`. Used for displaying
         appropriate error messages. Must be non-negative.
-
-    compat_args: OrderedDict
-        A ordered dictionary of keys and their associated default values.
+    compat_args : OrderedDict
```

Comment:
Is this still an OrderedDict? IIRC we removed all occurrences of those from code base recently

Reply:
Remove it from the doc?

Reply:
Hmm was going to say just label as `dict` but it looks like there is a detailed note about ordered requirements. Still should just be a dict but have to clean up the associated note; can do in a follow up if you'd like

Reply:
Looking at the detailed note, it's just to be compatible some buggy versions of numpy, and by looking at the git-blame of it, it way 4 years ago.

It's probably fixed by now, but I really cant know for sure, because there's no mention of a
specific/range of problematic numpy's versions.

Do you feel comfortable with removing the detailed note about the ```OrderedDict```?